src/components/scroll-up-button/ScrollUpButton.constants.ts

```diff
@@ -0,0 +1,5 @@
+export const SCROLL_UP = {
```

Comment:
It's private implementation details for this component. I think we don't need to create separate files for that. It can be inside component and not-exported so we're not leaking implementation details for other modules.